### ISSUE TITLE
trinity_api does not report consistent status 

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#578]
[root@controller(dmaster) ~]# /trinity/report 
**Version:** r7 85ff0e9
**Environment:** dmaster

```
[root@controller(dmaster) ~]# systemctl status trinity_api
● trinity_api.service - LSB: Trinity API server
   Loaded: loaded (/etc/rc.d/init.d/trinity_api)
   Active: inactive (dead)
     Docs: man:systemd-sysv-generator(8)
[root@controller(dmaster) ~]# service trinity_api status
5901
Running.
```

**Comment by themisa** [comment]
**Version:** r7_login_image 660733d
**Environment:** tmaster


And the opposite usually:
The "configuration" below is a broken trinity_api, since HPC dashboard returns errors. Usually after a controller reboot. 
```
[root@controller ~(keystone_a)]# service trinity_api status
Not Running.
[root@controller ~(keystone_a)]# systemctl status trinity_api
● trinity_api.service - LSB: Trinity API server
   Loaded: loaded (/etc/rc.d/init.d/trinity_api)
   Active: active (exited) since Thu 2016-01-14 12:29:04 UTC; 43min ago
     Docs: man:systemd-sysv-generator(8)
  Process: 1440 ExecStart=/etc/rc.d/init.d/trinity_api start (code=exited, status=0/SUCCESS)
   Memory: 0B

Jan 14 12:29:02 controller.cluster systemd[1]: Starting LSB: Trinity API server...
Jan 14 12:29:04 controller.cluster trinity_api[1440]: Starting trinity_api:[  OK  ]
Jan 14 12:29:04 controller.cluster systemd[1]: Started LSB: Trinity API server.
```

**Comment by hansthen** [comment]
May be fixed in 34a30eee.

**Comment by themisa** [comment]
Looks fixed:

[root@controller ~(keystone_a)]# /trinity/report
**Version:** r7 b55e0e1
**Environment:** tmaster
[root@controller ~(keystone_a)]# ^C
[root@controller ~(keystone_a)]# service trinity_api status
6514
Running.
[root@controller ~(keystone_a)]# systemctl status trinity_api 
● trinity_api.service - LSB: Trinity API server
   Loaded: loaded (/etc/rc.d/init.d/trinity_api)
   Active: active (running) since Wed 2016-01-20 12:39:16 UTC; 46min ago
     Docs: man:systemd-sysv-generator(8)
  Process: 6504 ExecStart=/etc/rc.d/init.d/trinity_api start (code=exited, status=0/SUCCESS)
   Memory: 19.0M
   CGroup: /system.slice/trinity_api.service
           6514 /usr/bin/python /usr/bin/trinity_api

Jan 20 12:39:14 controller.cluster systemd[1]: Starting LSB: Trinity API server...
Jan 20 12:39:16 controller.cluster trinity_api[6504]: Starting trinity_api:[  OK  ]
Jan 20 12:39:16 controller.cluster systemd[1]: Started LSB: Trinity API server.
